### DISCUSSION TITLE
🛫 Release 1.3.0

### DISCUSSION THREAD

**Discussion by valnoxy** [valnoxy/t480-oc#102] [Announcements]
## 🚧 Important note about macOS 14 Sonoma
This release include support for macOS 14 Sonoma.
However, there will be limitations in macOS Sonoma:
  - Sonoma no longer supports Broadcom cards. Long live Intel?
  - This macOS version will probably be the last one that will be compatible with the hardware of the T480 and similar. 
  - Sonoma is currently in beta. Install at your own risk (as always)!

![Bildschirmfoto 2023-06-12 um 22 19 40](https://github.com/valnoxy/t480-oc/assets/53713528/5d349b05-f16b-42cd-9880-707a18e0e865)

## Changes
The following kexts was removed:
   - AppleBacklightFixup (merged into WhateverGreen)
   
The following base / kexts was updated:
   - **Airportitlwm**: 2.2.0
   - **AppleALC**: 1.8.3
   - **CPUFriend**: 1.2.6
   - **HibernationFixup**: 1.4.9
   - **Lilu**: 1.6.6
   - **VoodooI2C**: 2.7.1
   - **VoodooPS2**: 2.3.3
   - **VoodooRMI**: 1.3.5
   - **WhateverGreen**: 1.6.5
   - **YogaSMC**: 1.5.3
   - **VirtualSMC**: 1.3.2
   - **OpenCore**: 0.9.3

## Issues assigned to this release
   - #57
   - #58 

<hr /><em>This discussion was created from the release <a href='https://github.com/valnoxy/t480-oc/releases/tag/v1.3.0'>🛫 Release 1.3.0</a>.</em>

**Comment by haidar47x** [comment]
What about the the wireless driver? I saw the author of Airportitlwm released an alpha a week ago to add support for macOS Sonoma. However, I was wondering whether the current Intel Wireless cards in T480 work with Sonoma.
